helix-core/src/main/java/org/apache/helix/HelixManagerProperty.java

```diff
@@ -80,27 +86,32 @@ public long getHealthReportLatency() {
     return _healthReportLatency;
   }
 
-  public RealmAwareZkClient.RealmAwareZkConnectionConfig getZkConnectionConfig() {
-    return _zkConnectionConfig;
+  public RealmAwareZkClient.RealmAwareZkConnectionConfig getRealmAwareZkConnectionConfig() {
+    return _realmAwareZkConnectionConfig;
   }
 
   public RealmAwareZkClient.RealmAwareZkClientConfig getZkClientConfig() {
     return _zkClientConfig;
   }
 
+  public HelixZkClient.ZkConnectionConfig getZkConnectionConfig() {
```

Comment:
since this is new property, you can rename this method, so that clients don't have to change.